### ISSUE TITLE
Example https-gke / gke-node-port example produce error

### ISSUE THREAD

**Issue by shinhf** [terraform-google-modules/terraform-google-lb-http#293]
### TL;DR

While running scripts in https://github.com/terraform-google-modules/terraform-google-lb-http/tree/v6.3.0/examples/https-gke/gke-node-port there is an error:

error expected [compute.instanceGroups,compute.regionInstanceGroups], got [compute.instanceGroupManagers] while running provisioner "local-exec"

### Expected behavior

Terraform script should be completed successfully.

### Observed behavior

error expected [compute.instanceGroups,compute.regionInstanceGroups], got [compute.instanceGroupManagers] while running provisioner "local-exec"

### Terraform Configuration

```hcl
Terraform v1.3.6
```

### Terraform Version

```sh
Terraform v1.3.6
on linux_amd64
+ provider registry.terraform.io/hashicorp/google v4.47.0
+ provider registry.terraform.io/hashicorp/kubernetes v2.16.1
+ provider registry.terraform.io/hashicorp/null v3.2.1
```

### Additional information

Old code:
```
resource "null_resource" "default" {

  provisioner "local-exec" {
    command = "gcloud compute instance-groups set-named-ports ${google_container_cluster.default.node_pool[0].instance_group_urls[0]} --named-ports=${var.port_name}:${var.node_port} --format=json"
  }
}
```

Corrected version:
```
resource "null_resource" "default" {
  provisioner "local-exec" {
    command = "gcloud compute instance-groups set-named-ports ${google_container_cluster.default.node_pool[0].managed_instance_group_urls[0]} --named-ports=${var.port_name}:${var.node_port} --format=json"
  }
}
```
